ExampleGallery/UAP/CameraEffectExample.xaml.cs

```diff
@@ -119,6 +126,17 @@ private void EffectCombo_SelectionChanged(object sender, SelectionChangedEventAr
             var task = SetEffect(effect);
         }
 
+        private void VariableEffectValueSlider_OnValueChanged(object sender, RangeBaseValueChangedEventArgs e)
+        {
+            if (sender == null || e == null || effectPropertySet == null) return;
+
+            //prevent changes smaller than .01 to update the effect
+            if (Math.Abs(e.OldValue - e.NewValue) < 0.1) return;
```

Comment:
What's the purpose of ignoring small changes?   Setting a value into a property bag is cheap - why not always report any change?